mediator/med_methods_mod.F90

```diff
@@ -455,10 +449,14 @@ subroutine med_methods_FB_init(FBout, flds_scalar_name, fieldNameList, FBgeom, S
              end if
 
              ! Determine ungridded lower and upper bounds for lfield
-             ungriddedCount=0  ! initialize in case it was not set
-             call ESMF_AttributeGet(lfield, name="UngriddedLBound", convention="NUOPC", &
-                  purpose="Instance", itemCount=ungriddedCount,  isPresent=isPresent, rc=rc)
```

Comment:
I am curious if you get some error message when the UngriddedLBound is not set? Will you still get ungrddedCount=0?

Reply:
I don't think so. In the update, if there is no UBound present, it will set the ungridded count to zero.